jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -20,6 +20,10 @@ mkdir -p "${LOGS_DIR}"
 mkdir -p "${LOGS_DIR}/manifests"
 cp -r /tmp/manifests/* "${LOGS_DIR}/manifests"
 
+# Fetch target cluster logs which were collected before re-pivoting
+mkdir -p "${LOGS_DIR}/target_cluster_logs"
+cp -r /tmp/target_cluster_logs/* "${LOGS_DIR}/target_cluster_logs"
```

Comment:
`"${LOGS_DIR}/target_cluster_logs"` should have trailing slash added for clarity as it is directory.

Reply:
done